draft-ietf-sframe-enc.md

```diff
@@ -54,47 +54,52 @@ informative:
 
 --- abstract
 
-This document describes the Secure Frame (SFrame) end-to-end encryption and authentication mechanism for media frames in a multiparty conference call, in which central media servers (SFUs) can access the media metadata needed to make forwarding decisions without having access to the actual media.
-The proposed mechanism differs from other approaches through its use of media frames as the encryptable unit, instead of individual RTP packets, which makes it more bandwidth efficient and also allows use with non-RTP transports.
+This document describes the Secure Frame (SFrame) end-to-end encryption and authentication mechanism for media frames in a multiparty conference call, in which central media servers (selective forwarding units or SFUs) can access the media metadata needed to make forwarding decisions without having access to the actual media.
+
+The proposed mechanism differs from the Secure Real-Time Protocol (SRTP) in that
+it is independent of RTP (thus compatible with non-RTP media transport) and can
+be applied to whole media frames in order to be more bandwidth efficient.
```

Comment:
This one is 80 char wrapped, not the other section.
We should probably try to be consistent.

Reply:
Updated the sections affected by this PR.  I can do a global 80-char-wrap PR once the current queue drains.